pkg/acsengine/const.go

```diff
@@ -32,7 +32,7 @@ const (
 )
 
 const (
-	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.3"
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.6.0-beta.1"
```

Comment:
@anhowe I started working on porting to the official 1.6 release. Please let me know if you guys are working on it as well to avoid duplicate effort.